routes/productRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+
+const productController = require("../controllers/productController");
+
+const router = express.Router();
+
+router.get("/", productController.getProductsByParameter);
```

Comment:
```suggestion
router.get("", productController.getProductsByParameter);
```
뒤에 주소가 더 추가되지 않는다면 `/`는 작성하지 말아주세요~